core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -597,39 +597,36 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
   }
 
 
-  setStartTime() {
+  setStartTime(val) {
     this.formFields.map((item) => {
       if (item.label === "TIME RANGE") {
         if (item.selected === "Last 24 Hours") {
-          this.startTime = moment().subtract(1, 'day').toISOString();
+          this.startTime = moment(val).subtract(1, 'day');
```

Comment:
do u need to call moment() here or can u just do `val.subtract(1, 'day');`?

Reply:
yes ,we need to call.

Reply:
for  environment overview it is working as expected.